spec/javascripts/govuk_publishing_components/analytics-ga4/gtm-click-tracking.spec.js

```diff
@@ -11,6 +11,7 @@ describe('Google Tag Manager click tracking', function () {
 
   afterEach(function () {
     document.body.removeChild(element)
+    window.location.hash = ''
```

Comment:
I can't envision a scenario in testing where we would want the hash to persist across tests so I think this is fine.